spec/json_spec.rb

```diff
@@ -14,5 +14,13 @@
   it 'should response with specific serializer' do
     get '/with-specific-serializer/'
     match_response_schema(:test_with_specific_serializer)
+
+    json = JSON.parse last_response.body
+    expect(json['bar']).to eq('This is a custom serializer - Test')
+  end
+
+  it 'should response with each serializer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.